policyengine_us/variables/gov/irs/credits/ctc/refundable/ctc_social_security_excess.py

```diff
@@ -0,0 +1,22 @@
+from policyengine_us.model_api import *
+
+
+class ctc_social_security_excess(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Refundable Child Tax Credit Social Security Excess"
+    unit = USD
+    documentation = (
+        "The excess amount in social security for child tax credit calculation"
+    )
+    definition_period = YEAR
+    reference = "https://www.law.cornell.edu/uscode/text/26/24#d"
+
```

Comment:
use adds/subtracts instead of the formula

Reply:
i thought these were policy parameters? you can use those instead of lists in adds/subtracts

Reply:
@vrathi101 You can use the previously created parameters for the adds and subtracts instead of adding them in the variable